lib/docs/filters/padrino/entries.rb

```diff
@@ -0,0 +1,17 @@
+module Docs
+  class Padrino
+    class EntriesFilter < Docs::EntriesFilter
+      def get_name
+        node = at_css('h1')
+        result = node.content.strip
+        result << ' event' if type == 'Events'
+        result << '()' if node['class'].try(:include?, 'function')
+        result
+      end
+
+      def get_type
+        return 'Ruby'
```

Comment:
Ahh, didn't know about the `rackup` commands - it's already looking awesome, I totally love it. The `Class:` and `Module:` has been removed:
![no_class_module](https://cloud.githubusercontent.com/assets/264708/15801112/79b7fd92-2a8b-11e6-8b89-983a0c5e65a9.png)